packages/legend-application-studio/src/stores/LegendStudioClientSideStorage.ts

```diff
@@ -0,0 +1,20 @@
+/**
+ * Copyright (c) 2020-present, Goldman Sachs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export enum LEGEND_STUDIO_CLIENT_SIDE_STORAGE_KEY {
```

Comment:
Let's do something like this (also, rename this file) - see my explanation in `StorageService` file

```ts
export enum LEGEND_STUDIO_SETTINGS_KEY {
  EITOR_STRICT_MODE = 'editor.strictMode',
  EDITOR_WRAP_TEXT = 'editor.wrapText',
}
```